assets/bingo/lang/de_de.json

```
@@ -1,17 +1,286 @@
+	"bingo.lobby.tutorial.game_info.general.heading": "Allgemein",
+	"bingo.lobby.tutorial.game_info.general": "Alle Items müssen genauso erhalten werden, wie angezeigt. Alle Items, die gleich aussehen, zählen. (z.B. ein verzaubertes Buch benötigt keine spezifische Verzauberung, aber ein rotes Bett muss rot sein und nicht blau.)",
+	"bingo.lobby.tutorial.game_info.world_gen.heading": "Anpassungen Welt-Erzeugung",
+	"bingo.lobby.tutorial.game_info.world_gen": "Minecraft Bingo benutzt einige Anpassungen bei der Erzeugung.",
+	"bingo.lobby.tutorial.game_info.world_gen.tiny_biomes.heading": "Winzige Biome: ",
+	"bingo.lobby.tutorial.game_info.world_gen.tiny_biomes": "Biome sind kleiner, um die Erkundung zu erleichtern. Dadurch können auch andere Unterschiede im Biom-Layout auftreten.",
+	"bingo.lobby.tutorial.game_info.world_gen.more_dungeons.heading": "Mehr Verliese: ",
+	"bingo.lobby.tutorial.game_info.world_gen.more_dungeons": "Damit Verliese eine sinnvolle Option in einem schnellen Spiel sind, ist die Rate von Verliesen um den Faktor 12,5 erhöht. Das sollte es einfacher machen, sie bei jedem Durchlauf zu finden.",
+	"bingo.lobby.tutorial.game_info.world_gen.caves.heading": "Höhlen Anpassungen: ",
+	"bingo.lobby.tutorial.game_info.world_gen.caves": "Die Frequenz von Höhlen wurde leicht erhöht (von 1/7 in Vanilla auf 1/6).\n\nAußerdem gibt es keine überfluteten Höhlen in Ozeanen.",
+	"bingo.lobby.tutorial.game_info.world_gen.no_mineshafts.heading": "Keine Minen: ",
+	"bingo.lobby.tutorial.game_info.world_gen.no_mineshafts": "Minen haben sich als übermächtig gezeigt, da sie leichten Zufgriff auf viele Truhen geben. Minen sind daher ausgeschaltet in Bingo.",
+	"bingo.lobby.tutorial.game_info.world_gen.ites.heading": "Keine *ite: ",
+	"bingo.lobby.tutorial.game_info.world_gen.ites": "Alle Steinvarianten wie DiorIT, GranIT and AndesIT sind in dieser Version komplett entfernt.\nDadurch soll das Zumüllen des Inventars reduziert werden.",
+	"bingo.lobby.tutorial.game_info.world_gen.ambient.heading": "Umgebungsbeleuchtung: ",
+	"bingo.lobby.tutorial.game_info.world_gen.ambient": "Du kannst im Dunkeln besser sehen, aber immer noch hell von dunkel unterscheiden.",
+
+	"bingo.lobby.tutorial.get_tools.sign.line1": "Nach dem Start",
+	"bingo.lobby.tutorial.get_tools.sign.line2": "brauchst Du ",
+	"bingo.lobby.tutorial.get_tools.sign.line3": "erstmal ",
+	"bingo.lobby.tutorial.get_tools.sign.line4": "Werkzeug.",
+
+	"bingo.lobby.team_selection.sign.line1": "",
+	"bingo.lobby.team_selection.sign.line2": "Team",
+	"bingo.lobby.team_selection.sign.line3": "wechseln",
+	"bingo.lobby.team_selection.sign.line4": "",
+
+	"bingo.lobby.end.sign.line1": "Die Außenwelt",
+	"bingo.lobby.end.sign.line2": "",
+	"bingo.lobby.end.sign.line3": "Gefahr!",
+	"bingo.lobby.end.sign.line4": "Bleibt drinnen!",
```

```suggestion
	"bingo.lobby.end.sign.line4": "Bleib drinnen!",
```
```suggestion
	"bingo.lobby.end.sign.line4": "Drinnen Bleiben!",
```

```suggestion
	"bingo.lobby.end.sign.line4": "Drinnen Geblieben",
```
_Minor_ Just some suggestions with different amounts of humorous phrasing (last one doesn't fit with exclamation mark) :D